packages/base64/base64.js

```diff
@@ -1,22 +1,22 @@
 // Base 64 encoding
 
-var BASE_64_CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+const BASE_64_CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
 
-var BASE_64_VALS = {};
+const BASE_64_VALS = {};
```

Comment:
I don't think it's relevant here (and I know you didn't write the original) but, in "modernization", I have a strong preference for switching to `Object.create(null)` (rather than `{}`) to initiate empty objects which will be used as lookups from user-input since `Object`s created with `{}` (object initializer notation) inherit (non-"own") attributes from the global `Object.prototype`, such as `hasOwnProperty`, `valueOf`, etc.

In this case, the user-input appears to be limited to single characters at a time so `hasOwnProperty` would be out of the question, but if the global `Object`s prototype was manipulated elsewhere, values that we didn't expect could suddenly become part of this `BASE_64_VALS` hash:

```js
// Rogue code?
Object.prototype['-'] = 45;

// This code.
const BASE_64_CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
const BASE_64_VALS = {};
const getVal = ch => ch === '=' ? -1 : BASE_64_VALS[ch];

// Use of `getVal`
getVal("-"); // returns 45;
```

Though I'm not sure if that would _actually_ break `Base64.decode` below...

Reply:
I will adjust that and keep it in mind for other PRs!